README.md

```diff
@@ -112,7 +112,7 @@ Apprentices should memorize [**this**](Cheatsheet.md) within the first month.
 ### Week 4 - Intermediate JavaScript
 
 1. [HTML/JS Game Project](/projects/js-html-games.md)
-1. [jQuery](/web/jquery.md)
+1. [jQuery](/web/jquery.md) 
```

Comment:
```suggestion
1. [jQuery](/web/jquery.md)
```
Please find some valid typo 😄

Reply:
I thought we need to delete this link so typo doesn't matter here😅
So, we need to find existing typos? I thought just create one and in the next PR of same kind remove that.😌